test/integration/storage/test_files.rb

```diff
@@ -0,0 +1,134 @@
+require "helpers/integration_test_helper"
+
+class TestFiles < FogIntegrationTest
+  begin
+    client_email = Fog.credentials[:google_client_email]
+    @@connection = Fog::Storage::Google.new
+    @@connection.put_bucket("fog-smoke-test", options = { "acl" => [{ :entity => "user-" + client_email, :role => "OWNER" }] })
+    @@connection.put_bucket_acl("fog-smoke-test", :entity => "allUsers", :role => "READER")
+  rescue Exception => e
+    puts e
+    puts e.backtrace
+  end
+
+  begin
+    @@directory = @@connection.directories.get("fog-smoke-test")
+    @@connection.put_object("fog-smoke-test", "fog-testfile", "THISISATESTFILE")
+    @@file = @@directory.files.get("fog-testfile")
+  rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?